src/components/sdk/MeetingControls/AudioInputControl.tsx

```diff
@@ -17,11 +17,17 @@ interface Props {
   muteLabel?: string;
   /** The label that will be shown when microphone is unmuted, it defaults to `Unmute`. */
   unmuteLabel?: string;
+  /** Title attribute for the icon when muted, it defaults to `Muted microphone` */
+  mutedTitle?: string;
```

Comment:
Should we have "icon" in the props names here? Like `mutedIconTitle`, `unMutedIconTitle` and so on. This would help builders understand the prop more easily apart from the description.

Reply:
Yep, good suggestion, will make this change